packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -80,6 +89,7 @@ export default function PostOptionsMenu({
   const { trackEvent } = useContext(AnalyticsContext);
   const { hidePost, unhidePost } = useReportPost();
   const { openModal } = useLazyModal();
+  const { items } = useContext(ActiveFeedContext);
```

Comment:
Since we introduced this here, perhaps we should just abstract the `feedQueryKey` here as well.
Don't really see any point in prop drilling it now.

Reply:
`ActiveFeedContext` already has `queryKey` which is current key for active feed.

Reply:
It was changed to use they `queryKey` in [4464bda](https://github.com/dailydotdev/apps/pull/2318/commits/4464bdafbd5fe0368fe65303b9b8ff3199a711e0#diff-0eb690e81644b4b6b5fae075b498cec565153a663e9838c2e241f88e5e77f9aaR90)